hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/logic/ServicesTxnManager.java

```diff
@@ -94,6 +103,10 @@ public void process(TxnAccessor accessor, Instant consensusTime, long submitting
             ledger.begin();
 
             if (needToPublishMigrationRecords) {
+                if (bootstrapProperties.getBooleanProperty(PropertyNames.ACCOUNTS_BLOCKLIST_ENABLED)) {
```

Comment:
I understand what's going on but nevertheless seems odd to gate this by a boolean called "needToPublishMigrationRecords" since creating these accounts has nothing to do with needing to publish migration records.  But then why have _two separate_ "one time only" booleans.  Perhaps this is the time that this boolean should be renamed to be something more general, like "stuffToDoOneTimeOnly".

Reply:
See #5961 for pending tech debt